api/api/error_handler.py

```diff
@@ -55,6 +55,42 @@ def _cleanup_detail_field(detail: str) -> str:
     return ' '.join(str(detail).replace("\n\n", ". ").replace("\n", "").split())
 
 
+async def handle_expect_header(request: ConnexionRequest, exc: ContentSizeExceeded = None) -> ConnexionResponse:
+    """Handler for 'Expect' header.
+    
+    Parameters
+    ----------
+    request : ConnexionRequest
+        Incoming request.
+    exc : ContentSizeExceeded
+        Raised exception if content size exceeds the limit.
+
+    Returns
+    -------
+    Response
+        HTTP Response returned to the client.
+    """
+    problem = {
+       "title": "error : 417",
+       "detail": "Expectation Failed."
+    }
+
+    if "Expect" in request.headers:
```

Comment:
Use single quotes, same for the rest of the file

```suggestion
    if 'Expect' in request.headers:
```